modules/nutanix-csi-driver-reqs.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/preparing-to-install-nutanix.adoc
+
+:_content-type: CONCEPT
+[id="nutanix-csi-driver-reqs_{context}"]
+= Nutanix CSI Driver Operator requirements
+
+The following requirements must be met in order to install the CSI Driver Operator:
+
+* <To do>
+* <To do>
+* <To do>
```

Comment:
Given that storage is day-2 item, is this section even applicable? If it is applicable, I suspect we would need to move the requirements into the Post installation section.

Reply:
Yes, I think we can cover this in the "Configuring the default storage container" section.

Reply:
Removed the section.